crowdin.yml

```diff
@@ -0,0 +1,6 @@
+preserve_hierarchy: true
+files:
+  # /content/{vodeVersion}/en-US/doc/{filesInSubFolders}
+  # markdown extensions - md, mkd, mkdn
+  - source: /content/v*/en-US/doc/**/.(m?d?|markdown)
```

Comment:
`/content/v*/en-US/doc/**/*.md` should do the trick here. `md` appears to be the only extension used in the node docs.